exercises/concept/intro-select/intro-select.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE all_data AS
```

Comment:
Isn't there supposed to be a `.meta/example.sql` file with a working solution?

Reply:
It's optional.  Do you think one is needed?

Reply:
Sorry. An exemplar for concept exercises. The exemplar is required, not optional.

Reply:
Done.